tracker/index.js

```diff
@@ -99,11 +80,52 @@ import { removeTrailingSlash } from '../lib/url';
     });
   };
 
-  /* Start */
 
-  pageView();
+  const scheduledCalls = window.umami.calls;
+  window.umami = {};
+  window.umami.collect = (type, params, uuid) => {
```

Comment:
You don't need to use `window.umami` everywhere. Just use the local function names then declare an object at the end.

Reply:
check